app/src/main/res/values/colors.xml

```diff
@@ -36,5 +36,6 @@
   <color name="translucent_background">#52000000</color>
 
   <color name="search_query_highlight">#52FFC800</color>
+  <color name="overdue_section_header_grey">#6C737A</color>
```

Comment:
I think we can just use the `color_on_surface_67` instead of defining a new colour? Can be changed in a later PR.